src/viewer/mobile/Panel.tsx

```diff
@@ -220,6 +227,23 @@ class Panel extends React.Component<Props & TwitchExtProps, State> {
 			boardState && boardState.game_type === BnetGameType.BGT_BATTLEGROUNDS;
 		const emptyDeck = deck && deck.cards && !deck.cards.length;
 
+		const isHidden = (feature: Feature) =>
+			hasFeature(
+				this.props.config.hidden && !isNaN(+this.props.config.hidden)
+					? +this.props.config.hidden
+					: 0,
+				feature,
+			);
+
+		const gameType =
+			boardState && boardState.game_type
+				? boardState.game_type
+				: BnetGameType.BGT_UNKNOWN;
+
+		const hideBobsBuddy = isHidden(Feature.BOBSBUDDY);
+
+		const hideDeckList = isHidden(Feature.DECKLIST);
```

Comment:
These should be changed to
```
const showBobsBuddy = !isHidden(Feature.BOBSBUDDY);
const showDeckList = !isHidden(Feature.DECKLIST);
```
to have less negation and cleaner reading code below (`showBobsBuddy ? <BobsBuddy/> : null` rather than `!hideBodyBuddy ? <BobsBuddy/> : null`)